bin/phpunit/test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const { spawnSync } = require( 'child_process' );
+
+/**
+ * Internal dependencies
+ */
+const { plugins } = require( '../../plugins.json' );
+const { formats } = require( '../plugin/lib/logger' );
+
+const args = process.argv.slice( 2 );
+const { WPP_PLUGIN, WPP_MULTISITE } = process.env;
+const pluginBasePath = path.resolve( __dirname, '../../' );
+const pluginBaseName = path.basename( pluginBasePath );
+const pluginsDir = path.resolve( pluginBasePath, 'plugins' );
+const phpunitBin = path.resolve( pluginBasePath, 'vendor/bin/phpunit' );
+const wpEnvBin = path.resolve( pluginBasePath, 'node_modules/.bin/wp-env' );
+const wpPluginsDir = `/var/www/html/wp-content/plugins/${ pluginBaseName }`;
+
+const _plugins = []; // Store absolute paths to plugins.
+
+if ( WPP_PLUGIN ) {
```

Comment:
@thelovekesh Unrelated to the ongoing discussion in #1012: If we implement new JS tooling scripts like this, I think they should be incorporated into the existing foundation in `bin/plugin`. It uses `commander` for a full CLI suite, which has some benefits like built-in help / documentation functionality and error handling infrastructure. This could be reused rather than implementing standalone commands like this.